client/checkout/blocks/woopay.js

```diff
@@ -16,7 +16,11 @@ const WooPay = ( { isStatic, api } ) => {
 		} );
```

Comment:
This could also be done by adding the event object param to onClick and calling `event.preventDefault();`, yeah? Any advantage to doing that instead of removing the inherited "submit" button type? Either way, this is working for me. Thanks for this!

Reply:
I am seeing that basically all of the tests are failling with this message:

`Fatal error: Uncaught Error: Call to undefined function getallheaders() in /home/runner/work/woocommerce-payments/woocommerce-payments/woocommerce-payments.php on line 172`

Do we need to add a Polyfill for this function to WCPay? 🤔

Reply:
Looks like this was fixed by #3481 👍